snapper/AppUtil.cc

```diff
@@ -223,7 +223,7 @@ namespace snapper
     string
     stringerror(int errnum)
     {
-#if (_POSIX_C_SOURCE >= 200112L) && ! _GNU_SOURCE
+#if (_POSIX_C_SOURCE >= 200112L) && ! _GNU_SOURCE || MUSL_LIBC
```

Comment:
This requires packagers to define `MUSL_LIBC` when building snapper, since musl doesn't define it itself.